automated-comments/ruby/general/explicit_return.md

```diff
@@ -0,0 +1,13 @@
+Remove the explict return to simplify the method.
+
+In Ruby, a method will return the object that was returned from the last evaluated statement. Explicitly using the `return` keyword is therefore unnecessary unless we are returning early in a method.
+
+For example, this will print `16` as the `multiply` method will return the result of `a * b`, it's final expression.
```

Comment:
```suggestion
For example, this will print `16` as the `multiply` method returns the result of `a * b`, its final expression.
```

I think that's meant to be just possessive?

Reply:
Ridiculous language ;)